components/expenses/ExpenseFormPayeeStep.js

```diff
@@ -225,6 +228,31 @@ const ExpenseFormPayeeStep = ({
               </StyledInputField>
             )}
           </Field>
+          {values.payee?.legalName && (
+            <StyledInputField
+              label={
+                <React.Fragment>
+                  <FormattedMessage defaultMessage="Legal name" />
```

Comment:
I'm surprised that `react-intl` doesn't screen here, isn't `id` mandatory?

Reply:
Looking at the language file, it generates one automatically. Interesting!

Here we'll want to re-use the existing string though.

![image](https://user-images.githubusercontent.com/1556356/133742379-6f1e9d32-c3f1-4467-95f6-bf2afba436cb.png)

Reply:
Glad we discovered that we don't need the `id`. But I've change this to re-use the existing string. 😉